src/index.ts

```diff
@@ -25,6 +27,41 @@ app.get('/', (req, res) => {
   res.send('Hello world');
 });
 
+// add result to a specified motivator
+app.post('/motivator/result/:motivatorId', (req, res) => {
+  log.info('Adding a result to a motivator');
+
+  if (!req.body) {
+    res.status(422).send('Request body undefined');
```

Comment:
```suggestion
    res.status(422).send({ error: 'Request body undefined'});
```
API stick required `422 Unprocessable entity` errors to return an object with the string property `error` containing the message.